packages/asc-ui/src/components/Alert/AlertStyle.ts

```diff
@@ -4,7 +4,13 @@ import { svgFill, themeColor, themeSpacing } from '../../utils'
 import Button from '../Button'
 import Heading from '../Heading'
 
-export type Level = 'normal' | 'attention' | 'warning' | 'error'
+export type Level =
+  | 'normal'
+  | 'attention'
+  | 'warning'
+  | 'error'
+  | 'valid'
+  | 'focus'
```

Comment:
I think I'd prefer to use terminology here that represents the state rather than the color. Besides that I think we need to discuss which variants should be supported because this is getting kinda out of hand.